apps/main/src/lend/components/PageLoanCreate/Page.tsx

```diff
@@ -61,11 +61,11 @@ const Page = () => {
   })
 
   useEffect(() => {
-    if (isSuccess && !market) {
+    if (!market) {
```

Comment:
Hmm, I wonder if we should do this? Technically there's no `loading` state anymore right? Or is this page still shown when not yet hydrated? There's either a market or not, there's no in between state anymore.

Regardless I'd rather show either a loading state or error state rather than redirect

Reply:
Okay I need to check if hydrated otherwise it instantly redirects back lol